docs/ldml/tr35.md

```diff
@@ -2904,17 +2920,67 @@ The syntax characters are listed in the table below:
 |  }   | U+007D | RIGHT CURLY BRACKET  | Strings in set; Perl property syntax       |
 |      | U+0020 U+0009..U+000D U+0085<br/>U+200E U+200F<br/>U+2028 U+2029 | ASCII whitespace,<br/>LRM, RLM,<br/>LINE/PARAGRAPH SEPARATOR | Ignored except when escaped |
 
+Note that some syntax characters only have a special meaning in a certain context. In particular:
+* Out of all above syntax characters, only \\, \}, and whitespace have a special meaning inside strings (**\[\{\[a-z\]\}\]** is the set of the string '\[a-z\]', **\[\{\$blah\}\]** is the set of the string '\$blah').
+* \$ is equivalent to \uFFFF when appearing at the very end of a set with or without trailing whitespace (**[a-z\$]**, **[a-z\$ ]**), and used as starting indicator for a variable reference elsewhere, in which case the variable name will be the longest match on the `var` nonterminal (**[\$my_set]**).
```

Comment:
> on the var nonterminal ([$my_set]).
This is a bit mysterious

Reply:
Which part is mysterious? (In this section I've followed the convention that all special cases have a UnicodeSet example in parentheses that makes use of that special case)

Reply:
I think it would be clearer in this instance to say (such as [...)